packages/noports_core/lib/src/sshnp/util/sshnpd_channel/sshnpd_channel.dart

```diff
@@ -87,16 +87,16 @@ abstract class SshnpdChannel with AsyncInitialization, AtClientBindings {
   Future<SshnpdAck> handleSshnpdPayload(AtNotification notification);
 
   /// Wait until we've received an acknowledgement from the daemon.
-  /// Returns true if the deamon acknowledged our request.
+  /// Returns true if the daemon acknowledged our request.
   /// Returns false if a timeout occurred.
-  Future<SshnpdAck> waitForDaemonResponse() async {
+  Future<SshnpdAck> waitForDaemonResponse({int maxWaitMillis = 15000}) async {
     // Timer to timeout after 10 Secs or after the Ack of connected/Errors
```

Comment:
This comment is out of sync with the changes, maybe change it to say
```dart
/// Timer to timeout after [maxWaitMillis] milliseconds or after the Ack of connected/Errors
```

Reply:
Done